HIP/hip-1238.md

```diff
@@ -0,0 +1,219 @@
+---
+hip: 1238
+title: Network Quiescence
+author: Lazar Petrović <@lpetrovic05>, Kelly Greco <@poulok>, Mark Blackman <@Mark-Swirlds>
+working-group: Richard Bair (@rbair23)
+requested-by: Hashgraph
+type: Standards Track
+category: Core
+needs-hiero-review: Yes
+needs-hedera-approval: Yes
+hedera-review-date:
+hedera-approval-status:
+status: Review
+created: 2025-07-11
+discussions-to: https://github.com/hiero-ledger/hiero-consensus-node/pull/20069
+updated: 2025-07-15
+---
+
+## Abstract
+
+The Quiescence feature introduces a mechanism to pause event creation in Hiero networks during periods of no transaction
+activity, optimizing resource usage in low-volume networks. By halting event generation when all user transactions have
+reached consensus and are in fully signed blocks, Quiescence minimizes network traffic, computational overhead, and
+long-term storage of event data and system transactions. This feature is particularly advantageous for private or
+low-traffic Hiero networks, reducing operational and reoccurring long term storage costs. Quiescence maintains
+compatibility with existing consensus mechanisms, ensuring no impact on network security, performance, or functionality.
+This HIP details the quiescence conditions, mechanisms for pausing and resuming event creation, and implementation
+considerations, including impacts on mirror nodes and SDKs. By prioritizing efficiency across the transaction lifecycle,
+Quiescence aligns with Hiero’s commitment to sustainable, efficient distributed ledger technology.
+
+## Motivation
+
+Current Hiero mainnet specifications mandate continuous event creation and gossip, even in the absence of transaction
+activity, to maintain consensus and network state. While robust for consistent network traffic, this approach is
+inefficient for networks with periodic usage where prolonged periods without transactions are common.
+
+The constant generation of events and blocks in such scenarios produces unnecessary data resulting in increased
+bandwidth, CPU, and long-term storage usage. These inefficiencies lead to increased operational costs and unnecessary
+resource utilization, misaligning with the needs of lightweight or intermittently active networks. By enabling the
+network to pause event creation when no non-ancient, non-consensus transactions are pending and fully signed blocks are
+achieved, the Quiescence feature addresses these shortcomings.
+
+Quiescence provides a cost-effective mode of operation for Hiero networks with intermittent usage.
+
+## Rationale
+
+The rationale fleshes out the specification by describing why particular design decisions were made. It should describe
+alternate designs that were considered and related work, e.g. how the feature is supported in other ecosystems. The
+rationale should provide evidence of consensus within the community and discuss important objections or concerns raised
```

Comment:
The Rationale section seems to be a description of what the rationale is supposed to contain(possibly a leftover from the HIP template). Can we update this section?